### PR TITLE
Resolves DCP2-477: ArcLight - Display call number instead of Collecti…

### PR DESCRIPTION
**Pull request by gkostin1966**
…on ID
clements
bentley

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -365,7 +365,10 @@ def raw
                                                 document.languages.present?
                                               }
 
-    config.add_summary_field 'eadid', label: 'Collection ID', accessor: :eadid
+    config.add_summary_field 'collection_unitid_ssm', label: 'Collection ID', accessor: :collection_unitid,
```
Comment (respinos) [inline app/controllers/catalog_controller.rb:368]: @cmkpowell should this label be "collection id" or "call number"?

Comment (cmkpowell) [reply app/controllers/catalog_controller.rb:368]: @respinos Call number

Comment (respinos) [reply app/controllers/catalog_controller.rb:368]: @gkostin1966 once the label has been updated to "Call Number", merge away.

File: app/views/catalog/show_actions/_collection_info.html.erb [hunk 2]
```diff
@@ -4,24 +4,19 @@
...
-        <%# if document.unitid.present? %>
-<!--          <tr>-->
-<!--            <td>Collection ID</td>-->
-<!--            <td><%#= document.unitid %></td>-->
-<!--          </tr>-->
-        <%# end %>
-
+        <% if /^\s*bhl\s*$/i.match?(document.repository_id) %>
         <tr>
           <td>Collection ID</td>
```
Comment (respinos) [inline app/views/catalog/show_actions/_collection_info.html.erb:9]: Update this label to "Call Number"

**Review by respinos — APPROVED** [review]

**Pull request merged**